SYCL/ESIMD/lsc/lsc_load2d_compare.cpp

```diff
@@ -0,0 +1,103 @@
+//==------------ load2d_compare.cpp - DPC++ ESIMD on-device test ----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc || esimd_emulator
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// The tests makes sure old and new load_2d API produce identical results.
+
+#include <iostream>
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/sycl.hpp>
+
+using namespace sycl;
+using namespace __ESIMD_NS;
+using namespace __ESIMD_ENS;
```

Comment:
Please do not use internal names in E2E tests.
```suggestion
using namespace sycl::ext::intel::esimd;
using namespace sycl::ext::intel::experimental::esimd;
```

Reply:
Fixed